biome.json

```diff
@@ -28,7 +28,7 @@
     "enabled": true,
     "indentWidth": 2,
     "indentStyle": "space",
-    "lineWidth": 80
+    "lineWidth": 150
```

Comment:
Might be best to keep it at 80 for the public repo

Reply:
done